### PR TITLE
PW-5129 Added paymentResult, enable TransactionPaymentTrigger

### PR DESCRIPTION
**Pull request by maassenbas**
## Tested scenarios

**Fixed issue**:

### CODE REVIEW

File: force-app/main/default/classes/BatchCapturePayment.cls [hunk 1]
```diff
@@ -54,6 +53,7 @@ global with sharing class BatchCapturePayment implements Database.Batchable<sObj
                         listToUpsert.add(tp);
                     }
                 } catch (Exception ex) {
+                    System.debug(ex.getMessage());
```
Comment (wboereboom) [inline force-app/main/default/classes/BatchCapturePayment.cls:56]: Maybe add a human readable string to this error so you can trace it when it shows in the logs

File: force-app/main/default/classes/TransactionPaymentHandler.cls [hunk 2]
```diff
@@ -0,0 +1,16 @@
+public with sharing class TransactionPaymentHandler {
```
Comment (wboereboom) [inline force-app/main/default/classes/TransactionPaymentHandler.cls:1]: what is a `with sharing` class?

**Review by wboereboom — APPROVED** [review]
🍕

File: force-app/main/default/classes/TransactionPaymentHandler.cls (hunk 2 above)
Comment (ShubhamVijaivargiya) [reply force-app/main/default/classes/TransactionPaymentHandler.cls:1]: Explicitly setting this keyword ensures that Apex code runs in the current user context. This is kind of a security measure in Salesforce.

File: force-app/main/default/classes/BatchCapturePayment.cls [hunk 3]
```diff
@@ -28,11 +28,10 @@ global with sharing class BatchCapturePayment implements Database.Batchable<sObj
 
                     if (mapStoreNameMerchant.containsKey(tp.ccrz__Storefront__c)) {
                         merchant = mapStoreNameMerchant.get(tp.ccrz__Storefront__c);
+
                     } else {
                         Boolean useCache = !Test.isRunningTest();
```
Comment (ShubhamVijaivargiya) [inline force-app/main/default/classes/BatchCapturePayment.cls:33]: Do we need useCache anymore?

**Review by ShubhamVijaivargiya — APPROVED** [review]
Shared feedback on BatchCapturePayment class, otherwise looks good.

File: force-app/main/default/classes/BatchCapturePayment.cls (hunk 3 above)
Comment (maassenbas) [reply force-app/main/default/classes/BatchCapturePayment.cls:33]: You are right, will delete it

File: force-app/main/default/classes/BatchCapturePayment.cls (hunk 1 above)
Comment (maassenbas) [reply force-app/main/default/classes/BatchCapturePayment.cls:56]: Will add a message there as well

**Pull request merged**